### PR TITLE
Translation of display strings in Tamil is not complete Fixes #10472

### LINKED ISSUE #10472: Tamil translation is not complete
- [ x] I have searched open and closed issues for duplicates
- [ x] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
All display text is not translated to Tamil
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
expected results will be submitted as a PR since it is too many to point out

### Device info

**Device:** Manufacturer Model XVI
**Android version:** 0.0.0
**Signal version:** 0.0.0

### PR DESCRIPTION
**Pull request by vairam-svs**
Translation of display strings in Tamil is not complete Fixes #10472

### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [ ] I have tested my contribution on these devices:
 * Device A, Android X.Y.Z
 * Device B, Android Z.Y
 * Virtual device W, Android Y.Y.Z
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### Description

- Signal word is not "transliterated" as name in Tamil, while in other languages that show 100% in transifex is transliterated
- Are the following texts needs to stay in English:
   * Signal needs permission to your microphone and camera.
   * Signal version: 
   * Signal package: 
   * Signal settings

This is change is not 100% complete. 
Also who is the owner /contributor for Tamil Language
Also, I am unable to choose Tamil language in transifex translation to JOIN

### CODE REVIEW

**Comment by vairam-svs** [comment]
I am submitting translation here as mentioned here https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md#translations because that tool does not allows me to choose Tamil (ta) to join team even though it has translation and its an available language already

**Review by Ashok314 — COMMENTED** [review]
Language translations are not to be made an issue or a PR here. You can contribute via transifex [see here](https://github.com/signalapp/Signal-Android#contributing-translations) 

About the changes in the file,
"Signal" is not to be translated into other languages. 
FYI refer to other translations.

**Comment by vairam-svs** [comment]
@Ashok314 see my comments about issues with transifex

File: app/src/main/res/values-ta/strings.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <string name="AbstractNotificationBuilder_new_message">புதிய செய்தி</string>
   <!--AlbumThumbnailView-->
   <!--ApplicationMigrationActivity-->
-  <string name="ApplicationMigrationActivity__signal_is_updating">Signal புதுப்பிக்கிறது …</string>
```
Comment (Ashok314) [inline app/src/main/res/values-ta/strings.xml:13]: ### Do not need to translate "Signal"
Should leave "Signal" as it is and translate others into another language. 
Same for the "Signal" appearing on other lines.

**Review by Ashok314 — CHANGES_REQUESTED** [review]
see other language translations as a reference.

**Review by Ashok314 — COMMENTED** [review]
This PR is unnecessary/invalid and should be closed. 
Thanks.

**Comment by alan-signal** [comment]
Thank you but translations are done only by our community of volunteer translators on a separate service to github. If we merged them here they would only be overwritten.

Please leave feedback on the [community forum](https://community.signalusers.org/c/translation-feedback/27). Our community of volunteer translators are able to view your suggestions, provide feedback on those independent changes, and carry over changes to our translation tools. We appreciate your help.

**Pull request closed without merging**